spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
OK. I'll move it to its own commit, wait until it's merged, and push back after a rebase.